ClosedXML.Examples/ClosedXML.Examples.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1;net40;net46</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net461</TargetFrameworks>
```

Comment:
net461 is EOL (Apr 26, 2022), would it be preferable to target supported frameworks, being `net462` ?

Reply:
@jafin These are internal projects, not libraries released to the public. I'd prefer to set it to the lower possible target in order to ensure everything works, tests pass, etc. Even though net461 is EOL, it's not as if Microsoft is going to come to users' PCs and physically delete the framework.

Reply:
@jahav @Pankraty Your opinion on keeping the internal projects on net461? net461 vs net462 is a minor issue to me, so we can increase it if there is enough support. I'd say once my default Visual Studio installation on a new PC doesn't have net461 anymore, I'd be inclined to increase it. I wouldn't want to install custom frameworks just to get ClosedXML to build.

Reply:
I would go for the lowest version, i.e. net461. It is most likely to show any problems, since it is earliest one. I agree with you.

Reply:
> It is most likely to show any problems

If `net461` is, so is `net462` too. 

> I wouldn't want to install custom frameworks just to get ClosedXML to build.

I want to vote for `net462` but this is the point which convinces me finally. Currently keeping `net461` is at least acceptable 🤣 .